addons/binding/org.openhab.binding.loxone/README.md

```diff
@@ -0,0 +1,245 @@
+
+  * Make sure your Miniserver is up and running and on the same network segment as openHAB server.
+  * Add Loxone binding from the available `Add-ons`.
+  * In `Configuration/System` page, set `Item Linking` to `Simple Mode` (don't forget to save your choice).
+  * Add your Miniserver Thing from the `Inbox`, after automatic discovery is performed by the framework during binding initialization.
+  * Configure your Miniserver by editing Miniserver Thing in `Configuration/Things` page and providing user name and password.
+  * Miniserver Thing should go online. Channels and Items will be automatically created and configured.
+  * On the `Control` page, you can test Miniserver Items and iteract with them.
+  * As the user interface, you may use [HABPanel](http://docs.openhab.org/addons/uis/habpanel/readme.html), where all Miniserver's items are ready for picking up, using entirely the graphical user interface.
+
+## Manual Configuration Example
+
+A more advanced setup requires manual creation and editing of openHAB configuration files, according to the instructions provided in [configuration user guide](http://docs.openhab.org/configuration/index.html).
+In this example we will manually configure:
+
+  * A Miniserver with serial number 504F2414780F, available at IP 192.168.0.220 and with web services port 80
+  * A Miniserver's user named "kryten" and password "jmc2017"
+  * Items for:
+      * Temperature of the Miniserver - a Virtual Analog State functional block
+      * State of a garage door - a Virtual Digital State funtional block (ON=door open, OFF=door closed)
+      * Kitchen lights switch - a Switch Subcontrol at the AI1 output of a Lighting Controller functional block (with a tag recognizable by Alexa service)
+      * Pushbutton to switch all lights off - a Virtual Input of Pushbutton type functional block (pushbutton realized by adding `autoupdate="false"` parameter)
+      * Kitchen blinds - a Jalousie functional block
+      * Lighting scene - a Lighting Controller functional block
+      * Output valve selection for garden watering - 8x Radio Button functional block, where only one valve can be open at a time
+      * A text displaying current alarm's state - a State functional block
+
+### things/loxone.things:
+
+```
+loxone:miniserver:504F2414780F [ user="kryten", password="jmc2017", host="192.168.0.220", port=80 ]
+```
+
+### items/loxone.items:
+
+```
+// Type       ID              Label                                  Icon          Tags         Settings
+
+Number        Miniserver_Temp "Miniserver temperature: [%.1f °C]"    <temperature>              {channel="loxone:miniserver:504F2414780F:0F2F2133-017D-3C82-FFFF203EB0C34B9E"}
+Switch        Garage_Door     "Garage door [MAP(garagedoor.map):%s]" <garagedoor>               {channel="loxone:miniserver:504F2414780F:0F2F2133-017D-3C82-FFFF203EB0C34B9E"}
```

Comment:
Couldn't there be a Contact channel type offered, so that you wouldn't need a mapping file? Or is this actually an "active" functionality that also accepts commands to open and close the garage door?